cadt-ui/templates/cadt.local.conf.j2

```diff
@@ -0,0 +1,57 @@
+server {
+    listen 80;
+    listen [::]:80;
+
+    server_name {{ certbot_domains | join( " " ) }};
+
+    root /dev/null;
+
+    include /etc/nginx/snippets/lets-encrypt-webroot.conf;
+
+    location / {
+        rewrite (.*) {{ cadt_ui_host }}$1 permanent;
+    }
+}
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    root /var/www/{{ cadt_ui_host }}/build;
+
+    server_name {{ certbot_domains | join( " " ) }};
+
+    ssl_certificate /etc/letsencrypt/live/{{ certbot_domains | first }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ certbot_domains | first }}/privkey.pem;
+
+    ssl_protocols             TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers               ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+    ssl_buffer_size           8k;
+    ssl_session_cache         shared:SSL:50m;
+    ssl_session_timeout       1d;
+    ssl_session_tickets       off;
```

Comment:
I think you can get these (or at least most of them) from this https://github.com/Chia-Network/ansible-roles/blob/main/certbot-cert/tasks/main.yml#L77:L78

Which is the latest best practice recommendations from lets encrypt

Reply:
Changed it - using the include now